src/modules/transaction/actions.ts

```diff
@@ -45,6 +46,58 @@ export type FetchTransactionFailureAction = ReturnType<
   typeof fetchTransactionFailure
 >
 
+// Fetch cross chain transaction
+
+export const FETCH_CROSS_CHAIN_TRANSACTION_REQUEST =
+  '[Request] Fetch Cross Transaction'
```

Comment:
```suggestion
  '[Request] Fetch Cross Chain Transaction'
```

Reply:
right?

Reply:
I've decided to remove the Cross Chain Transaction actions.